src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1078,6 +1088,16 @@ def get_active_query_names(self):
                 ret.append(query)
         return ret
 
+    def __dir__(self):
+        dir_list = set(list(self.__dict__.keys()) + dir(type(self)))
+        return dir_list - set(
+            [
+                child
+                for child in self.child_names + self.command_names + self.query_names
```

Comment:
Please also include `self.argument_names`

Reply:
Done, Thank you.